File: TP1/HelloWorldApp/app/src/main/java/fr/utbm/lo52/helloworldapp/MainActivity.java
package fr.utbm.lo52.helloworldapp;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void goToHelloWorld(View view)
    {
        Intent intent = new Intent(this, HelloWorld.class);
        startActivity(intent);
    }
}

File: TP1/HelloWorld/app/src/main/java/com/lo52/tp1/helloworld/MainActivity.kt
package com.lo52.tp1.helloworld

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun afficher(view : View){
        val intent = Intent(this, HelloWordActivity::class.java)
        startActivity(intent)
    }
}